neuro_evolution_ctrnn/brain_visualizer/position.py

```diff
@@ -0,0 +1,63 @@
+import networkx as nx
+import numpy as np
+
+class Positions():
+    def getGraphPositionsLean(self, w, h):
```

Comment:
Wenn ich das richtig verstehe greifst du in der `__init__.py` in Zeile 60 auf diese Methode zu und übergibtst das `PygameBrainVisualizer` Objekt richtig? Dann wäre es besser die Methodensignatur zu ändern auf
`getGraphPositionsLean(self, brain_visualizer, w, h)`. `self` ist im Allgemeinen in Python Konvention um auf die Attribute einer Klasse zuzugreifen